CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 ## [Unreleased][]
 
-[Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-aws/compare/0.13.0...HEAD
+[Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-aws/compare/0.14.0...HEAD
+
+## [0.14.0][]
```

Comment:
for next time, no need to set the version in the PR. We do this on release :)

that's okay for this one :)